bin/sync-db.sh

```diff
@@ -65,11 +65,21 @@ echo "$PG_DB_NAME"
 
 if [ $USE_LOCAL_DUMP -eq 0 ]; then
   if [ "${#PASSWORD}" -eq 0 ]; then
-    echo -n "Type in your LDAP password and press enter to continue: ";
+    echo -n "Type in your Sesame password and press enter to continue: ";
     read -s PASSWORD;
   fi
-  echo "Copying HPC $DISTANT_ENV $DISTANT_DUMP_NAME database to local using wget with username $USERNAME"
-  wget --http-user=$USERNAME --http-password=$PASSWORD https://snapshots.aws.ahconu.org/hpc/$DISTANT_ENV/$DISTANT_DUMP_NAME -O "$BACKUP_DIR/$DB_DUMP" || { echo 'Copying database from source failed' ; exit 1; }
+  echo "Obtaining an authentication token for snapshot access"
+  TOKEN=$(curl -s -XPOST https://auth.ahconu.org/oauth2/token -d "grant_type=password&client_id=token&username=${USERNAME}&password=${PASSWORD}" | jq '.access_token' | tr -d '"')
+
+  if [ -z "${TOKEN}" ]; then
+    echo "Unable to obtain an authentication token."
+    echo 
+    echo "Please check that your password is HTML encoded and that 'curl', 'jq' and 'tr' are installed in your \$PATH"
+    exit 1
+  fi
+
+  echo "Copying HPC $DISTANT_ENV $DISTANT_DUMP_NAME database to local using curl with token authentication"
+  curl -H "Authorization: Bearer ${TOKEN}" https://snapshots.aws.ahconu.org/hpc-sync/$DISTANT_ENV/$DISTANT_DUMP_NAME -o "$BACKUP_DIR/$DB_DUMP" || { echo 'Copying database from source failed' ; exit 1; }
```

Comment:
We switched from `curl` to `wget` because of its auto-retry during downloads. See https://github.com/UN-OCHA/hpc_service/pull/2046